controllers/partiesController.js

```diff
@@ -23,6 +23,22 @@ class PartiesController {
       data: [parties],
     });
   }
+
+  static show(req, res) {
```

Comment:
why do you name your function name as `show`

Reply:
Because the route that access this controller action is a `show` route for displaying a single party.